### DISCUSSION TITLE
Fabric 1.20 release

### DISCUSSION THREAD

**Discussion by TinyDerp** [plan-player-analytics/Plan#3060] [Q&A]
I don't expect it to come out any time soon, just because 1.20 just came out and all that, but still. When do you think the 1.20 release will come out. Depending what the answer I will know what my plans are. 
Of course if you don't know or are not sure when its fine, its just a question/thought of mine nothing more.

**Comment by AuroraLS3** [comment]
There is a build here https://github.com/plan-player-analytics/Plan/pull/3047 (click the green check of the last commit, go to the build and select Summary)

I don't have a PC until 11.6. so the full release will likely be then or shortly after

**Reply by AuroraLS3** [reply]
https://github.com/plan-player-analytics/Plan/actions/runs/5154528966
